package.json

```diff
@@ -41,7 +42,7 @@
     "expo-haptics": "~11.2.0",
     "expo-image-manipulator": "~10.3.1",
     "expo-image-picker": "~13.1.1",
-    "expo-linking": "^3.3.0",
+    "expo-linking": "~3.1.0",
```

Comment:
Is the downgrade intentional

Reply:
When I run expo I get this message: Some dependencies are incompatible with the installed expo package version:
 - expo-linking - expected version: ~3.1.0 - actual version installed: 3.3.0. I downgraded it in order to avoid breaking something